src/micro_http/src/common/headers.rs

```diff
@@ -231,7 +269,9 @@ impl Headers {
                     break;
                 }
                 match headers.parse_header_line(header_line.as_bytes()) {
-                    Ok(_) | Err(RequestError::UnsupportedHeader) => continue,
+                    Ok(_) | Err(RequestError::HeaderError(HttpHeaderError::InvalidValue(_, _))) => {
```

Comment:
Why match `InvalidValue` here ? From what I understand Unsupported features or values should be ignored and errors related to unsupported headers or values should be propagated.